src/main/java/Duke.java

```diff
@@ -1,10 +1,145 @@
+import java.util.*;
 public class Duke {
+    // Constants
+    final static String PARTITION = "*******************************************";
+    final static String EXIT_COMMAND = "bye";
```

Comment:
It would be better if you could create an enum list that contains the commands so you need no declare them one by one